apps/expo/src/app/createpost.tsx

```diff
@@ -112,14 +128,12 @@ function CreatePost() {
             <TouchableOpacity
               className="w-44"
               onPress={() => {
-                console.log("Created post");
-                back();
+                createPost();
               }}
             >
               <FilledButton
                 onClick={() => {
-                  console.log("Created post");
-                  back();
+                  createPost();
                 }}
```

Comment:
should the createPost be on both of these?

Reply:
One of these doesn't work so it doesn't really matter. I just added for consistency

Reply:
can you remove the one that doesn't work then -- I don't think that a touchable opacity is needed over this button, so it can probably be changed to a view

Reply:
It is needed cause the whole view is touchable without feedback to close the keyboard (the discussion is multiline so the return makes a new line, not close the keyboard). The touchable without feedback then makes all the buttons not work, unless wrapped with a touchable and put the action there. 🤔

Reply:
Unless you have other ideas 🙏

Reply:
the filled button is still a touchable opacity though? why does it need to be wrapped in another one?

Reply:
It just doesn't work for some mysterious reason. I can remove it and screen record when I get back home. 

I've tried to do a bunch of research but this seems to be the widely used solution.

Reply:
https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/77c756df-29c0-44a0-91eb-f0586ceb497d


Ignore the sizing, but I keep pressing the button and nothing happens when you remove touchable opacity.

Reply:
Like theoretically there has to be a better way cause this is so inefficient and jank